src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,51 +107,106 @@ const CrossPodWriteForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleCrossPodUpload} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="cross-upload-doc">Upload document to username: </label>
-          <br />
-          <br />
-          <input
-            id="cross-upload-doc"
-            size="25"
-            name="crossPodUpload"
-            value={username}
-            onChange={(e) => setUsername(e.target.value)}
-            placeholder={selectedUser.username}
-          />
-        </div>
-        <div style={formRowStyle}>
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleCrossPodUpload} autoComplete="off">
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            {/* Either way, also determine whether to update all other forms to that */}
+            <Typography htmlFor="cross-upload-doc">Upload document to username:</Typography>
```

Comment:
Same as 155-156, can remove these lines. The id attribute could also be removed since it's no longer in use without the htmlFor